businessRules/ideas.js

```diff
@@ -1,9 +1,71 @@
-const { getAllIdeas } = require('../models/ideas');
+const {
+    getAllIdeas,
+    createIdea,
+    updateIdea,
+    deleteIdea,
+} = require('../models/ideas');
 
 function getIdeas() {
     return getAllIdeas();
 }
 
+function sortedByDonationSumIdeas() {
+    const ideasList = getIdeas();
+
+    const sortedIdeasList = ideasList.sort(
+        (a, b) => a.totalDonationSum - b.totalDonationSum
```

Comment:
Try to avoid using single letter names. You can use `ideaA` and `ideaB` names here

Reply:
Updated and tested. )